### PR TITLE
LLD - log content cards clicks not working when no url passed

### PR DESCRIPTION
**Pull request by cgrellard-ledger**
### 📝 Description

LLD log content cards clicks wasn't working when no url was passed to the content card. We now set a dummy one when calling the logContentCardClick method of the braze SDK to bypass this
The following message appeared in the console when clicking on a content card and the event was not appearing in the Braze dashboard : `Braze: Card NjQ1MzZjZjEzMzg4ZWIxNjdmNDRjOTkyXyRfY2M9MWUyYmFjNmEtNjg5MS02MmEyLTQ3YjQtNThjYzRhYTVhNzgzJm12PTY0NTM2Y2YxMzM4OGViMTY3ZjQ0Yzk4YSZwaT1jbXA= has no url. Not logging click to Braze servers.`

Also removing the referralProgram card from the Carousel as it will now be handled by Braze

### ❓ Context

- **Impacted projects**: `ledger-live-desktop` 
- **Linked resource(s)**: [LIVE-7299] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-7299]: https://ledgerhq.atlassian.net/browse/LIVE-7299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/Carousel/helpers.tsx [hunk 1]
```diff
@@ -59,190 +42,53 @@ export const getTransitions = (transition: "slide" | "flip", reverse = false) =>
...
-      size: {
-        width: 10,
-        height: 13,
-      },
-    },
-  ],
-};
-
 export const useDefaultSlides = () => {
-  const referralProgramConfig = useFeature("referralProgramDesktopBanner");
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/src/renderer/components/Carousel/helpers.tsx:200]: Should be removed every where since there is no use of it?

**Review by mcayuelas-ledger — APPROVED** [review]

**Comment by gre** [comment]
👋  we just merged https://github.com/LedgerHQ/ledger-live/pull/3359 which greatly improved the TypeScript checks on Ledger Live Desktop projects (components, modals, screens) and effectively make it type check on the CI. Could you rebase with it and ensure there is no issue on `pnpm desktop typecheck`? Thanks and don't hesitate to ask for any TS help.

**Comment by desirendr** [comment]
👋 @gre I guess the rebase will happen on the PR done for develop, this one is for the current release 🙏

**Pull request merged**